libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -915,6 +944,17 @@ def Snapshot(self,
     self.BlockOperation(response, zone=self.zone)
     return GoogleComputeSnapshot(disk=self, name=snapshot_name)
 
+  def Delete(self) -> None:
+    """Delete a Disk."""
+
+    gce_disk_client = self.GceApi().disks()
+    request = gce_disk_client.delete(
+        project=self.project_id, disk=self.name, zone=self.zone)
+    response = request.execute()
+    logger.info(
+        self.FormatLogMessage('Deleted Disk: {0:s}'.format(self.name)))
+    return response
```

Comment:
remove the return or change the return type above, linter is complaining

Reply:
done